openmoc/plotter.py

```diff
@@ -208,7 +211,8 @@ def plot_segments(track_generator):
 # @param gridsize an optional number of grid cells for the plot
 # @param xlim optional list/tuple of the minimim/maximum x-coordinates
 # @param ylim optional list/tuple of the minimim/maximum y-coordinates
-def plot_materials(geometry, gridsize=250, xlim=None, ylim=None):
+# @param z_level optional the z level (default is Geometry's _z_level)
+def plot_materials(geometry, gridsize=250, xlim=None, ylim=None, z_level=None):
```

Comment:
If `z_level` is actually a coordinate rather than a level index, perhaps `zcoord` would be more appropriate? I won't make this comment in the code below since it looks like it `z_level` is the notation used everywhere right now.